cmd/opm/alpha/bundle/build.go

```diff
@@ -42,6 +42,7 @@ func newBundleBuildCmd() *cobra.Command {
 		* All manifests yaml must be in the same directory. 
         `,
 		RunE: buildFunc,
+		Args: cobra.NoArgs,
```

Comment:
Unrelated to your changes: that bundleBuildCmd long description looks largely misidented to me if you're looking for additional things to tackle.

Reply:
Yeah, just pulled these changes down and here's the outupt:

```bash
$ ./bin/opm alpha bundle --help
The "opm alpha bundle build" command will generate operator
        bundle metadata if needed and build bundle image with operator manifest
        and metadata for a specific version.

        For example: The command will generate annotations.yaml metadata plus
        Dockerfile for bundle image and then build a container image from
        provided operator bundle manifests generated metadata
        e.g. "quay.io/example/operator:v0.0.1".

        After the build process is completed, a container image would be built
        locally in docker and available to push to a container registry.

        $ opm alpha bundle build --directory /test/0.1.0/ --tag quay.io/example/operator:v0.1.0 \
		--package test-operator --channels stable,beta --default stable --overwrite

		Note:
		* Bundle image is not runnable.
		* All manifests yaml must be in the same directory.
```

Reply:
Good call out, I'll tidy that up while I'm at it.